### PR TITLE
Refine InstructorSearchPageAction slice

### PR DESCRIPTION
**Pull request by taniach**
Changed indentation
Moved constants to Const.java
Did not add new tests in ActionTest because testing is done as a browser test

### CODE REVIEW

File: src/main/java/teammates/ui/controller/InstructorSearchPageAction.java [hunk 1]
```diff
@@ -110,22 +107,30 @@ protected ActionResult execute() throws EntityDoesNotExistException {
...
             Iterator<FeedbackResponseAttributes> fr = frs.iterator();
+            
             while (fr.hasNext()) {
                 FeedbackResponseAttributes response = fr.next();
-                FeedbackQuestionAttributes relatedQuestion = getRelatedQuestionOfResponse(frCommentSearchResults, response);
-                InstructorAttributes instructor = this.getInstructorForCourseId(response.courseId, instructors);
+                FeedbackQuestionAttributes relatedQuestion = getRelatedQuestionOfResponse(
+                                                frCommentSearchResults, response);
+                InstructorAttributes instructor = this.getInstructorForCourseId(
+                                                response.courseId, instructors);
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/InstructorSearchPageAction.java]: No need to break this line.

File: src/main/java/teammates/ui/controller/InstructorSearchPageAction.java [hunk 2]
```diff
@@ -110,22 +107,30 @@ protected ActionResult execute() throws EntityDoesNotExistException {
...
         while (iterFr.hasNext()) {
             List<FeedbackResponseAttributes> frs = iterFr.next().getValue();
             Iterator<FeedbackResponseAttributes> fr = frs.iterator();
+            
             while (fr.hasNext()) {
                 FeedbackResponseAttributes response = fr.next();
-                FeedbackQuestionAttributes relatedQuestion = getRelatedQuestionOfResponse(frCommentSearchResults, response);
-                InstructorAttributes instructor = this.getInstructorForCourseId(response.courseId, instructors);
+                FeedbackQuestionAttributes relatedQuestion = getRelatedQuestionOfResponse(
+                                                frCommentSearchResults, response);
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/InstructorSearchPageAction.java]: Indent more, until 2nd line is on the right side of `=`?

File: src/main/java/teammates/ui/controller/InstructorSearchPageAction.java [hunk 3]
```diff
@@ -110,22 +107,30 @@ protected ActionResult execute() throws EntityDoesNotExistException {
...
+                InstructorAttributes instructor = this.getInstructorForCourseId(
+                                                response.courseId, instructors);
+                
                 boolean isVisibleResponse = true;
-                boolean needCheckPrivilege = relatedQuestion == null || !(relatedQuestion.recipientType == FeedbackParticipantType.NONE ||
-                        relatedQuestion.recipientType == FeedbackParticipantType.INSTRUCTORS ||
-                                relatedQuestion.recipientType == FeedbackParticipantType.STUDENTS);
+                boolean needCheckPrivilege = relatedQuestion == null || !(relatedQuestion.recipientType == FeedbackParticipantType.NONE
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/InstructorSearchPageAction.java]: Line too long (have to scroll in github diff view)

File: src/main/java/teammates/ui/controller/InstructorSearchPageAction.java [hunk 4]
```diff
@@ -110,22 +105,30 @@ protected ActionResult execute() throws EntityDoesNotExistException {
...
-                                relatedQuestion.recipientType == FeedbackParticipantType.STUDENTS);
-                boolean isNotAllowedForInstructor = instructor == null || !(instructor.isAllowedForPrivilege(response.giverSection,
-                        response.feedbackSessionName, Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS))
-                        || !(instructor.isAllowedForPrivilege(response.recipientSection,
-                                response.feedbackSessionName, Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
+                boolean needCheckPrivilege = relatedQuestion == null 
+                                              || !(relatedQuestion.recipientType == FeedbackParticipantType.NONE 
+                                                   || relatedQuestion.recipientType == FeedbackParticipantType.INSTRUCTORS 
+                                                   || relatedQuestion.recipientType == FeedbackParticipantType.STUDENTS);
+
```
Comment (kanghj) [inline src/main/java/teammates/ui/controller/InstructorSearchPageAction.java]: Can you help me remove this variable (needCheckPrivilege). I missed this out in #2982.

**Comment by kanghj** [comment]
Apart from that, it looks good to merge.

File: src/main/java/teammates/ui/controller/InstructorSearchPageAction.java (hunk 4 above)
Comment (taniach) [inline src/main/java/teammates/ui/controller/InstructorSearchPageAction.java]: You mean delete the whole statement?

**Pull request merged**